callbacks/review_callbacks.py

```diff
@@ -108,6 +135,7 @@ def update_review_status_and_return(value, status, selected_rows):
                 f"{str(len(reviewed_data))} Classifications reviewed",
                 False,
                 "",
+                {"token": latest_token, "token_expiration": latest_token_expiration},
```

Comment:
I see, are we passing it back and forth to the functions? Should we instead use some internal state that's accessible across the functions?